aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -733,19 +724,17 @@ module aptos_framework::stake {
         operator: &signer,
         pool_address: address,
         new_consensus_pubkey: vector<u8>,
-        proof_of_poseesion: vector<u8>,
     ) acquires StakePool, ValidatorConfig {
-        rotate_consensus_key_internal(operator, pool_address, new_consensus_pubkey, proof_of_poseesion, true);
+        rotate_consensus_key_internal(operator, pool_address, new_consensus_pubkey, true);
```

Comment:
Given that branching is `genesis` is not needed, perhaps the function should now go back to not receiving this parameter at all. @so-schen made this change to branch on `genesis`, so please check with him if this is ok.

Reply:
done https://github.com/Entropy-Foundation/aptos-core/pull/9/commits/3fefc82a2c107f84b89b654ccb012cbdd6c7d3a1

Reply:
agree